### ISSUE TITLE
Add separate, more detailed section for the Input Writer?

### ISSUE THREAD

**Issue by wilhadams** [theochem/iodata#254]
Adding things like non-standard basis set options (#221) are making it possible to do a lot of useful and complex things with the input writer, which is really exciting. However, when I was previously adding examples for using custom templates, I started to worry that the "Input Files" section of the getting started page was beginning to get pretty lengthy. Would it make sense to break out most of the input writer details into a new page, and leave just the basic usage in the getting started page? 

@tovrstra @FarnazH @PaulWAyers

**Comment by tovrstra** [comment]
Yes, most definitely. It would be good to have a separate section with more elaborate examples, while keeping the "Getting Started" is kept short and simple.

While doing this, it could be good to address #210 too. Executing the more elaborate examples int the unit tests would guarantee the examples keep working. (This also implies the necessary input files are present to run the examples, which would also be helpful.)
